app/routes/public/exhibition/index.js

```diff
@@ -20,7 +20,10 @@ export default class ExhibitionRoute extends Route {
         val  : 'accepted'
       }
     ];
+    const { success, token } = await this.loader.load(`/events/${eventDetails.id}/chat-token`);
```

Comment:
Wrong logic, first this is duplication of code. Secondly, this should be fetched in the component. That's why it is logging in user before opening the chat